hpc_plugin/tasks.py

```diff
@@ -55,6 +55,13 @@ def preconfigure_job(config,
     ctx.source.instance.runtime_properties['workdir'] = \
         ctx.target.instance.runtime_properties['workdir']
 
+    ctx.logger.info("credentials:" )
+    for key, value in config['credentials'].iteritems() :
+        if not 'password' in key:
```

Comment:
Should be `if key != 'password'`. Also other credentials as the private_key or private_key_password must be masked if present.

Apart from this, I think this should be more a debug log than an info one. In this regard, we should have a mechanism from the portal to decide if we want debug messages or not.